README.md

```diff
@@ -77,38 +78,73 @@ This sample app lives in this repo in the `sample` directory.
 
 ## Jetpack Compose support
 
-Radiography will automatically render Composables found in your view tree if the Compose Tooling
-library is on the classpath. If you're using Compose, this is likely already the case (the
+[Jetpack Compose](https://developer.android.com/jetpack/compose) is Google's new UI toolkit. It is
+currently in pre-release status, and changes frequently. However, if you're already experimenting
+with Compose in your apps, any UI that is defined in Compose will not show up as Android views.
```

Comment:
>  any UI that is defined in Compose will not show up as Android views.

this is a bit confusing, show up where exactly?

How about:

> any UI that is defned in Compose does not include any Android view.

Reply:
Agreed it's confusing. Technically a Compose UI _may_ [include android views](https://developer.android.com/jetpack/compose/interop#views-in-compose). This whole paragraph is awkward. How about replacing it with:

> [Jetpack Compose](https://developer.android.com/jetpack/compose) is Google's new declarative UI toolkit. It is a completely new implementation, and does not use Android views itself (although it [interoperates](https://developer.android.com/jetpack/compose/interop#views-in-compose) with them seamlessly).

Reply:
Done.